### PR TITLE
README update for the changes undergone

### PR DESCRIPTION
**Pull request by niccolopaganini**
1. Updated README like @JGreenlee mentioned to successfully build
2. Maintaining current version
3. Will update again after updating JDK in this intel machine

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -156,14 +161,6 @@ $ cp www/json/startupConfig.json.sample www/json/startupConfig.json
...
 ```
 
-### Activation (after install, and in every new shell)
-
-```
-$ source setup/activate_native.sh
-```
-
-### Activation (after install, and in every new shell)
-
```
Comment (shankari) [inline README.md:166]: why did you remove this? We do need to activate after install.

File: README.md [hunk 2]
```diff
@@ -174,10 +171,15 @@ If connecting to a development server over http, make sure to turn on http suppo
 
 ### Run in the emulator
 
+Pick a version and execute the following:
```
Comment (shankari) [inline README.md:174]: I don't know that "type of build" == "version"

```suggestion
Pick a type of build and execute the following:
```

You should also indicate how people can find the valid "type of build" supported.

**Review by shankari — CHANGES_REQUESTED** [review]
Please indicate "Testing done"
"If it is not tested, it is broken".

In particular, I would like to see if the instructions work without activation (see comment below)

File: README.md (hunk 1 above)
Comment (niccolopaganini) [reply README.md:166]: I was able to build the app successfully. Let me verify the above again just to be sure.

Comment (shankari) [reply README.md:166]: were you able to build the app starting in a new terminal and following the instructions in the README exactly? 
When you check, please show, don't tell.

**Comment by niccolopaganini** [comment]
Ugh.. I lost all changes I had made to the README while pushing the changes to my repo. I will work on the README once I get some rest. Apologies (not sure what happened)

**Comment by niccolopaganini** [comment]
opened a new PR #1037
